dependencies.py

```diff
@@ -206,26 +270,60 @@ def _ensure_prerequisite_is_installed(prerequisite="pip"):
         raise RuntimeError(msg)
 
 
-def _dependencies_target_dir(our_dir=OUR_DIR):
-    """Return python dir inside our profile
+def _dependencies_target_dir(*, our_dir=OUR_DIR, create=False) -> Path:
```

Comment:
Waar was die `*` nou voor? Ik dacht dat ik het al gevraagd had, maar da's denk ik in een andere PR die je hierin gemerged hebt ofzo? Beetje dubbel werk voor m'n gevoel :-)